strongbox-user-management/src/main/resources/etc/conf/security-users.xml

```diff
@@ -27,7 +27,62 @@
             <encryption-algorithm>plain</encryption-algorithm>
         </credentials>
         <roles>
-            <role>USER_ROLE</role>
+            <role>ADMIN</role>
         </roles>
-    </user>    
+    </user>
+    <user>
+        <username>deployer</username>
+        <credentials>
+            <password>password</password>
+            <encryption-algorithm>plain</encryption-algorithm>
+        </credentials>
+        <roles>
+            <role>UI_MANAGER</role>
+        </roles>
+
+        <features>
+            <storages>
+                <storage id="storage0">
+                    <repositories>
+                        <repository id="act-releases-1">
```

Comment:
The most important part here is to be able to further control and fine-grain the access for the users based on paths in the repository. It should be possible to specify something like this here:

                                  <granted-paths>
                                       <path>com/carlspring/foo</path>
                                       <path>org/carlspring/foo</path>
                                  </granted-paths>